core/src/main/java/com/radixdlt/p2p/transport/handshake/AuthInitiateMessage.java

```diff
@@ -95,34 +95,47 @@ public final class AuthInitiateMessage extends BaseHandshakeMessage {
   @DsonOutput(DsonOutput.Output.ALL)
   private final HashCode nonce;
 
-  @JsonProperty("networkId")
+  @JsonProperty("nid")
   @DsonOutput(DsonOutput.Output.ALL)
-  private final int networkId;
+  private final byte networkId;
+
+  @JsonProperty("networkVersion")
```

Comment:
isn't this more of a "protocolVersion"? or am I misinterpreting?

Reply:
hmh, I think of it as a higher-level network version, not just the protocol